mteb/tasks/Reranking/multilingual/WikipediaRerankingMultilingual.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import MultilingualTask
+from ....abstasks.AbsTaskReranking import AbsTaskReranking
```

Comment:
```suggestion
from mteb.abstasks. MultilingualTask import MultilingualTask
from mteb.abstasks.AbsTaskReranking import AbsTaskReranking
```

I believe the issue is that you were importing the module "MultilingualTask" instead of the object "MultilingualTask". This issue arises from having the exact same name.